### PR TITLE
Merging effective 2.1 release to develop.

### PR DESCRIPTION
**Pull request by rHorsey**
This gem remains backwards compatible with the analysis-spreadsheet and v2 AMIs. Once https://github.com/NREL/OpenStudio-analysis-spreadsheet/pull/24 has been merged, this should be as well.

### CODE REVIEW

**Comment by rHorsey** [comment]
Note - the recent AMI flush killed a lot of these tests - I'll fix them before the merge.

**Comment by rHorsey** [comment]
FYI @nllong I'm not sure what to do about the rest of the tests - what determines stable, and how can we get a stable AMI tag so we can update the tests to point to it?

File: lib/openstudio/aws/aws.rb [hunk 1]
```diff
@@ -244,7 +246,8 @@ def create_workers(number_of_instances, options = {}, user_id = 'unknown_user')
 
         logger.info 'Waiting for server/worker configurations'
 
-        @os_aws.configure_server_and_workers
+        return @os_aws.configure_server_and_workers unless @dockerized
+        @os_aws.configure_swarm_cluster if @dockerized
```
Comment (nllong) [inline lib/openstudio/aws/aws.rb]: if / else block this.

File: lib/openstudio/lib/openstudio_aws_instance.rb [hunk 2]
```diff
@@ -95,7 +95,7 @@ def create_and_attach_volume(size, instance_id, availability_zone)
       volume_id: resp[:volume_id],
       instance_id: instance_id,
       # required
-      device: '/dev/sdh'
+      device: '/dev/sdm'
```
Comment (nllong) [inline lib/openstudio/lib/openstudio_aws_instance.rb]: is this always going to be true. I've seen sda/sdh... isn't this based on the instance type?

File: lib/openstudio/lib/openstudio_aws_instance.rb [hunk 3]
```diff
@@ -381,7 +385,7 @@ def wait_command(command)
                 logger.info("wait_command #{command} is true")
                 flag = 1
               else
-                sleep 10
+                sleep 1
```
Comment (nllong) [inline lib/openstudio/lib/openstudio_aws_instance.rb]: seems fast, maybe 3?

**Comment by rHorsey** [comment]
Review completed and unsupported tests removed.

**Pull request merged**